english/crochet/crochet.md

```diff
@@ -0,0 +1,3 @@
+Crochet teddy bears are the cutest.  You should start working on one today.  
+
+Click here to [return to reality] (https://github.com/udacity/create-your-own-adventure/blob/master/english/marshmallow.md).
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.